src/components/App.jsx

```diff
@@ -102,16 +102,21 @@ class App extends Component {
     this.fetchDiscussions()
   }
 
+  renderCardContainer() {
```

Comment:
I wonder if it would be more readable if we pulled out the other two ternarys into methods like this?